rusk/Cargo.toml

```diff
@@ -2,11 +2,11 @@
 name = "rusk"
 version = "0.2.0"
 authors = [
-  "CPerezz <carlos@dusk.network>", 
-  "zer0 <matteo@dusk.network>", 
-  "Victor Lopez <victor@dusk.network>",
-  "Jules de Smit <jules@dusk.network>",
-  "Luke Pearson <luke@dusk.network>"
+    "CPerezz <carlos@dusk.network>",
+    "zer0 <matteo@dusk.network>",
+    "Victor Lopez <victor@dusk.network>",
+    "Jules de Smit <jules@dusk.network>",
+    "Luke Pearson <luke@dusk.network>"
```

Comment:
I already started to remove the `authors` field in cargo in some of the crates, since now it's optional. We might want to do the same here too – especially because mostly everyone is working on Rusk now, and this field either will become outdated or huge.